src/components/RGBColorPicker.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+class RGBColorPicker extends Component {
```

Comment:
Well, here you need another component with state inside to make it work as it should and from here (the parent componet) you'll pass props to that component.